chart/openfaas/README.md

```diff
@@ -67,12 +56,20 @@ Now deploy OpenFaaS from the helm chart repo:
 helm repo update \
  && helm upgrade openfaas --install openfaas/openfaas \
     --namespace openfaas  \
-    --set basic_auth=true \
-    --set functionNamespace=openfaas-fn
+    --set functionNamespace=openfaas-fn \
+    --set basicAuthAutogen=true 
 ```
 
 > The above command will also update your helm repo to pull in any new releases.
 
+Retrieve the OpenFaaS credentials with:
+
+```sh
+USER=$(kubectl -n openfaas get secret basic-auth -o jsonpath="{.data.basic-auth-user}" | base64 --decode) && \
```

Comment:
The user will always be admin, so let's drop the user bit.

Reply:
Done